README.md

```diff
@@ -20,97 +20,96 @@ ytdl('http://www.youtube.com/watch?v=A02s8omM_hI')
+* `format` - Primarily used to download specific video or audio streams. This can be a specific `format` object returned from `getInfo`.
+  * Supplying this option will ignore the `filter` and `quality` options since the format is explicitly provided.
+* `range` - A byte range in the form `{start: INT, end: INT}` that specifies part of the file to download, ie {start: 10355705, end: 12452856}.
+  * This downloads a portion of the file, and not a separately spliced video.
+* `begin` - What time in the video to begin. Supports formats `00:00:00.000`, `0ms, 0s, 0m, 0h`, or number of milliseconds. Example: `1:30`, `05:10.123`, `10m30s`. For live videos, this also accepts a unix timestamp or Date, and defaults to `Date.now()`.
+  * This option may not work on super short (less than 30s) videos, and has to be at or above 6s, see [#129](https://github.com/fent/node-ytdl-core/issues/129).
+  * It may also not work for some formats, see [#219](https://github.com/fent/node-ytdl-core/issues/219).
 * `liveBuffer` - How much time buffer to use for live videos in milliseconds. Default is `20000`.
 * `requestOptions` - Anything to merge into the request options which [miniget](https://github.com/fent/node-miniget) is called with, such as headers.
 * `highWaterMark` - How much of the video download to buffer into memory. See [node's docs](https://nodejs.org/api/stream.html#stream_constructor_new_stream_writable_options) for more.
 * `retries` - The number of retries ytdl is allowed to do before terminating the stream with an error. The default is set to 5.
-* `lang` - The 2 character symbol of a language. Is by default set to "en".
+* `lang` - The 2 character symbol of a language. Default is `en`.
 
 ```js
 // Example with `filter` option.
 ytdl(url, { filter: (format) => format.container === 'mp4' })
   .pipe(fs.createWriteStream('video.mp4'));
 ```
 
-#### Event: 'info'
-* `Object` - Info.
-* `Object` - Format.
+#### Event: info
+* [`ytdl.videoInfo`](example/info.json) - Info.
+* [`ytdl.videoFormat`](typings/index.d.ts#L22) - Video Format.
 
-Emitted when the a video's `info` hash is fetched. Along with the chosen format metadata to download. `format.url` might be different if `start` was given.
+Emitted when the a video's `info` hash is fetched, along with the chosen format metadata to download. `format.url` might be different if `start` was given.
 
-An example of what Info and format may look like is in the [example folder](example/info.json).
+#### Event: response
+* [`http.ServerResponse`](https://nodejs.org/api/http.html#http_class_http_serverresponse) - Response.
 
-#### Event: 'response'
-* `http.ServerResponse` - Response.
+Emitted when the video response has been found and has started downloading. Can be used to get the size of the download. This is not emitted if there is an error with the download or it needs to reconnect to YouTube, although it is emitted after a successful reconnect.
```

Comment:
Just the part of the last sentence is fine and can be moved to the first sentence, no need to say when it's not emitted.

    Emitted when the video response has been found and has started downloading, and after any successful reconnect.

Reply:
Ok I am about to sleep but I'll make the change tomorrow :)